vllm/entrypoints/openai/serving_chat.py

```diff
@@ -506,7 +516,8 @@ async def chat_completion_full_generator(
             model=model_name,
             choices=choices,
             usage=usage,
-        )
+            prompt_logprobs=final_res.prompt_logprobs
```

Comment:
Why not just pass `final_res.prompt_logprobs` directly?

Reply:
good catch, artifact from before the default was back to None